### ISSUE TITLE
Ief Setup App does not support Azure Government

### ISSUE THREAD

**Issue by zachChilders** [azure-ad-b2c/Scripts#10]
Hello,

I'm trying to configure a SignUpSignIn flow via this article and getting the "Invalid Username or Password" error.  I've tried manually going over my configuration many times and the error persists.  

I see a lot of advice to configure via the Custom Policy tool here: https://b2ciefsetupapp.azurewebsites.net/
However, I am creating custom policies in a Government B2c tenant and the tool immediately fails because my domain is not in `onmicrosoft.com` but instead `onmicrosoft.us`

**Comment by KrishanPatel001** [comment]
Did you ever find a solution to this?

**Comment by zachChilders** [comment]
No, and based on that and several other factors we sought Auth services elsewhere.
